code/modules/uplink/uplink_items.dm

```diff
@@ -2145,6 +2194,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
 	restricted_roles = list(JOB_NAME_MEDICALDOCTOR, JOB_NAME_CHIEFMEDICALOFFICER, JOB_NAME_ROBOTICIST, JOB_NAME_PARAMEDIC, JOB_NAME_BRIGPHYSICIAN)
 	cost = 3
 
+/datum/uplink_item/role_restricted/brainwash_disk
+	name = "Brainwashing Surgery Program"
+	desc = "A disk containing the procedure to perform a brainwashing surgery, allowing you to implant an objective onto a target. \
+	Insert into an Operating Console to enable the procedure."
+	item = /obj/item/disk/surgery/brainwashing
+	cost = 2
+	restricted_roles = list(JOB_NAME_MEDICALDOCTOR, JOB_NAME_CHIEFMEDICALOFFICER)
```

Comment:
Brainwashing surgery does not need reducing to 2 TC, and should not be reduced specifically for medical doctors.

Reply:
Brainwashing surgery is incredibly difficult to pull of given you need an operating computer, a willing (or unwilling subject), tools and time in order to brainwash them. At the very least it should be lowered from costing 25% (5 TC) of your TC.

Reply:
Especially when the instant brainwashing hypnoflash is two TC extra with none of the previously mentioned downsides.

Reply:
Markus, you've clearly missed the meta when Brainwashing was non-stop thanks to incursions, the mechanic to de-brainwash being easier, the implant that allowed to do surgery and not to mention the abductors old brainwashing method

It was very unfun, unbalanced and it's finally back to ''normal'' it does not need a lower TC and it shouldn't be only affecting doctors, it's extremely easy to pull a brainwash surgery, you literally hit them with a basic RP of anesthesia and if not you have the healthscanner (kinda bullshit) insta sleep, sleepy pen and others, not to mention **You can't detect that they've been brainwashed and it requires a LOBOTOMY** the Hypnoflash is two TC more cheaper because you can spot it with a health scanner, it does not erquire a lobotomy(iirc) and you can be fucked if someone says something different, ruining your directive, Brainwashing disk is also super meh and stealthy given they always deconstruct the machine and hide the disk, we should not go back again to the hell of brainwashing 300 people, SPECIALLY if it's still undetectable and STILL requires a lobotomy.

Reply:
It sounds to me that brainwashing needs more signs / feedback of their affliction. I've previously played around with the idea of making hypno brainwashed victims drool, moan and so on, to seem more "brain dead" while the surgery brainwash has the health scanner pop up.

Reply:
> basic RP of anesthesia
Do note that the usage of anesthesia is the exception rather than the rule, and is easily metagameable or simply disregarded.

Reply:
> It sounds to me that brainwashing needs more signs / feedback of their affliction. I've previously played around with the idea of making hypno brainwashed victims drool, moan and so on, to seem more "brain dead" while the surgery brainwash has the health scanner pop up.

Maybe, but in my opinion you need to come up with something like you've mentioned or just leave it be for now, trust me that meta of brainwashing was... horrible to play against and we're still suffering from all those pr's (no way of telling who's been brainwashed, fixing requires lobotomy which i've seen make players.. just cryo because they RNG'd the very very bad permanent traumas and so on)